code/_onclick/telekinesis.dm

```diff
@@ -144,8 +144,8 @@ var/const/tk_maxrange = 15
 		apply_focus_overlay()
 		focus.throw_at(target, 10, 1,user)
 		last_throw = world.time
-		user.changeNext_move(CLICK_CD_MELEE)
```

Comment:
I'm against removing the click delay. If you want you can lower it though, https://github.com/HippieStation/HippieStation13/blob/42915e2f0fc726df31f1cebd7f3206907862c391/code/__DEFINES/combat.dm#L49
I'd suggest CLICK_CD_RANGE or CLICK_CD_RAPID